api/routes/api.js

```diff
@@ -61,14 +54,12 @@ router.get('/regVoter/sendOTP', fetchVoter, async (req, res, next) => {
   } else {
     res.status(200).json({
       msg: 'Wrong phone number :(',
-      phonenumber: req.phone,
-      data,
     });
   }
 });
 
 // Verify OTP
-router.post('/regVoter/verifyOTP', async (req, res, next) => {
```

Comment:
any particular reason for removing async?

Reply:
As we are not awaiting any promises using the await keyword. We are using the .then syntax so I removed it as it served no purpose.